src/plugins/recordTypes/collectionobject/forms/default.jsx

```diff
@@ -23,6 +23,11 @@ const template = (configContext) => {
         <Row>
           <Col>
             <Field name="objectNumber" />
+            <Field name="objectCountGroupList">
+              <Field name="objectCountGroup" tabular={false}>
+                <Field name="objectCount" label="" />
+              </Field>
+            </Field>
```

Comment:
Here's the trick. We just want to render a field that displays the `objectCount` from the _first_ `objectCountGroup`, without rendering anything for the intervening fields. This is how to do that:

```
<Field name="objectCount" subpath={["ns2:collectionobjects_common", "objectCountGroupList", "objectCountGroup", "0"]} />
```

The `subpath` prop refers to the path under the enclosing field (in this case, `<Field name="document">`) in which to find this field. It effectively specifies intervening fields that you want to skip over rendering.